src/parser.y

```diff
@@ -418,10 +418,9 @@ const_def
 
 %%
 
-void yyerror(YYLTYPE * loc, struct kafel_ctxt* ctxt, yyscan_t scanner,
-             const char *msg) {
+void yyerror(YYLTYPE * loc, struct kafel_ctxt* ctxt,
+             __attribute__((unused)) yyscan_t scanner, const char *msg) {
```

Comment:
just drop the parameter name instead

Reply:
I believe that only works in C++. Leaving out the name in C results in `error: parameter name omitted`.

Reply:
Oh, right... This is C :-)
`(void)scanner;` might be a bit more portable, but the attribute is fine too here I guess.

Reply:
Better portability is always a plus and it is also what the YYUSE macro did. I have updated the PR to implement your suggestion. :)